src/Input/Select/Select.test.tsx

```diff
@@ -3,7 +3,7 @@ import * as React from 'react';
 import * as lodash from 'lodash';
 import * as renderer from 'react-test-renderer';
 import '@testing-library/jest-dom/extend-expect';
-import { fireEvent, render } from '@testing-library/react';
+import { fireEvent, render, getAllByTestId } from '@testing-library/react';
```

Comment:
eslint warning: 'getAllByTestId' is defined but never used

Reply:
Btw, would you mind to resolve this eslint error within this PR? 😅 
I think it occurred due to the upgrade of dependencies.

```
/src/Display/Slider/Slider.tsx
  151:3  error  componentWillReceiveProps is deprecated since React 16.9.0, use UNSAFE_componentWillReceiveProps instead, see https://reactjs.org/docs/react-component.html#unsafe_componentwillreceiveprops. Use https://github.com/reactjs/react-codemod#rename-unsafe-lifecycles to automatically update your components  react/no-deprecated
```

Reply:
>eslint warning: 'getAllByTestId' is defined but never used
Removed.

> Btw, would you mind to resolve this eslint error within this PR?
Changed to `UNSAFE_componentWillReceiveProps`. The component will eventually be converted to a functional component, which will replace this deprecated lifecycle method.